L/libjulia/common.jl

```diff
@@ -4,6 +4,11 @@ using BinaryBuilder, Pkg
 
 include("../../fancy_toys.jl") # for get_addable_spec
 
+# list of supported Julia versions
+if ! @isdefined julia_versions
+    julia_versions = [v"1.6.3", v"1.7", v"1.8", v"1.9", v"1.10", v"1.11"]
```

Comment:
Actually, does anyone remember why we start the list with `1.6.3` instead of `1.6`? Maybe should add a comment explaining that (or change to 1.6)

Reply:
I think you explain this here, I haven't tested if changing this to 1.6 breaks anything: https://github.com/JuliaPackaging/Yggdrasil/pull/3639